conduit/plugins/exporters/filewriter/file_exporter_test.go

```diff
@@ -21,29 +20,45 @@ import (
 	sdk "github.com/algorand/go-algorand-sdk/v2/types"
 )
 
+const (
+	defaultEncodingFormat = MessagepackFormat
+	defaultIsGzip         = true
+)
+
```

Comment:
These don't seem to be used. I think the defaults derived from `filewriter.FilePattern`.
```suggestion
```

Reply:
These are used below in the (repurposed) unit test `TestPatternDefault` and in the admittedly silly'ish `TestDefaults` which is just a sanity check for the future in case the defaults change.